index.bs

```diff
@@ -375,21 +375,23 @@ All fields following the sample entry 4CC are expressed as double digit decimals
 
 The profile parameter value, represented by a single digit decimal, SHALL equal the value of seq_profile in the [=Sequence Header=].
 
-The level parameter value SHALL equal the value of level in the [=Sequence Header=] for one of the operating point.
+The still parameter value, represented by a single digit decimal, SHALL equal the value of still_picture in the [=Sequence Header=].
+
+The level parameter value SHALL equal the first level value in the [=Sequence Header=].
 
 The bitDepth parameter value SHALL equal the value of BitDepth variable as defined in [[AV1]] derived from the [=Sequence Header=].
 
 The monochrome parameter value, represented by a single digit decimal, SHALL equal the value of mono_chrome in the [=Sequence Header=].
 
 The chromaSubsampling parameter value, represented by a three-digit decimal, SHALL have its first digit equal to subsampling_x and its second digit equal to subsampling_y and the third digit equal to chroma_sample_position, if the first two values are non-zero, or 0 otheriwise.
 
-The colourPrimaries, transferCharacteristics, matrixCoefficients and videoFullRangeFlag parameter values SHALL equal the value of matching fields in the [=Sequence Header=], if color_description_present_flag is set to 1, otherwise the fields can take any value.
+The colourPrimaries, transferCharacteristics, matrixCoefficients and videoFullRangeFlag parameter values SHALL equal the value of matching fields in the [=Sequence Header=], if color_description_present_flag is set to 1, otherwise they SHOULD not be set, defaulting to the values below. The videoFullRangeFlag is represented by a single digit.
 
-For example, codecs="av01.2.01.10.0.112.09.16.09.01" represents AV1 profile 2, level 1, 10-bit YUV content, non-monochrome, with 4:2:0 chroma subsampling, ITU-R BT.2020 primaries, ST 2084 EOTF, ITU-R BT.2020 non-constant luminance color matrix, and full-range chroma/luma encoding.
+For example, codecs="av01.2.0.01.10.0.112.09.16.09.1" represents AV1 profile 2, level 1, 10-bit YUV content, non-monochrome, with 4:2:0 chroma subsampling, ITU-R BT.2020 primaries, ST 2084 EOTF, ITU-R BT.2020 non-constant luminance color matrix, and full-range chroma/luma encoding.
```

Comment:
The example text needs an update:

AV1 profile 2, non-still, level 1,

Reply:
updated.